CONTRIBUTING.md

```diff
@@ -1,91 +0,0 @@
-# Contributing guide
```

Comment:
Why this file was removed?

Reply:
Do you think you can merge `sphinx/contributor_guide.rst` in `CONTRIBUTING.md`, and use this file in sphinx?

Reply:
in sklearn CONTRIBUTING.md links to the docs. this is not a bad thing to me.

Reply:
My previous comment that I don't know if got sent:
> I put back the `CONTRIBUTING.md` file and in it I added a link to the documentation

Reply:
I'm looking into this https://github.com/sphinx-doc/sphinx/issues/7000

Reply:
Done

Reply:
Nice, well done!